pkg/proxy/ipvs/proxier_test.go

```diff
@@ -434,27 +417,21 @@ func TestGetNodeIPs(t *testing.T) {
 		{
 			isIPv6:       true,
 			devAddresses: map[string][]string{"ipvs0": {"1.2.3.4", "1000::"}, "lo": {"127.0.0.1", "::1"}, "encap7": {"10.20.30.31", "2000::", "fe80::200:ff:fe01:1"}, "kube-ipvs0": {"1.2.3.4", "2.3.4.5", "2000::"}},
-			expectIPs:    []string{"1000::"},
+			expectIPs:    []string{"1000::", "2000::"},
```

Comment:
So the behavior has changed... well, ok, except it changes back again in the last commit... this needs to be cleaned up

Reply:
I would like to squash and force-push. Comment below...